subprojects/common/report-viewer/src/main/kotlin/com/avito/report/model/RunId.kt

```diff
@@ -3,22 +3,43 @@ package com.avito.report.model
 import com.avito.android.Result
 
 /**
- * Универсальный runId для идентификации сьюта тестов
- * - build config teamcity домешан, чтобы отличать прогон на одном API на develop
- *  (там так же как в релизе нет targetBranch) от buildRelease
+ * Universal identifier for test suits
+ * @param prefix optional prefix
+ * @param commitHash unique SHA identifier of commit
+ * @param buildTypeId is used to distinguish runs with the same API on `develop` branch
  */
-data class RunId(val commitHash: String, val buildTypeId: String) {
+data class RunId(
+    val prefix: String = "",
```

Comment:
we don't need a default value

Reply:
Why? With default value, we are able to create a new instance with both `RunId(hash, type)` and `RunId(prefix, hash, type)`. It's convenient, considering that `prefix` is optional.